app/src/test/java/com/davidtomas/taskyapp/features/auth/data/remote/request/RegisterRequestMother.kt

```diff
@@ -0,0 +1,11 @@
+package com.davidtomas.taskyapp.features.auth.data.remote.request
+
+object RegisterRequestMother {
+    private val base = RegisterRequest(
+        fullName = "fullName",
+        email = "email",
+        password = "password",
+    )
+
+    fun mock() = base
+}
```

Comment:
What is the purpose of this? This is not a mock and also doesn't add any value I feel like..

Reply:
I am used to do this, to reuse this mock in different tests. In case I need an specific mock I create a mockWithEmptyEmail for example that is a copy of the base one. I find it very useful in order to always reuse and dont create too many instances for all the tests

Reply:
Sure, but why this wrapper? Why not just an extension or so?
```
fun registerRequest(): RegisterRequest {
    return RegisterRequest(...)
}
```
I don't have a problem with having such pre-created classes, but just with the naming and structure of this class. Firstly, I find `Mother` isn't a very expressive way to say that this generates test instances and `mock()` isn't actually true, since it just returns a stub (so the name should be `stub()`).
A mock records calls to its functions which is definitely not the case here :D